docs/getting_started/optimizing_tasks.md

```diff
@@ -110,9 +110,9 @@ scale our workload:
 
 ```{code-cell} ipython3
 import math
-from typing import Tuple
+from typing import Tuple, List
 
-from flytekit import map_task
+from flytekit import map_task, task, workflow
```

Comment:
I believe these have been imported at the beginning for a different example.

Reply:
Do we want to make each example "self-contained"?  

While it's true that previous examples have the import, if a user goes directly to the Map Tasks section, the example won't be complete